playbooks/_bootstrap-common.yaml

```diff
@@ -1,41 +1,34 @@
 # vim:ts=2:sw=2:et:ai:sts=2
 
-# Playbook: Common bootstrapping roles.
+# Playbook: common bootstrapping roles.
 #
-# * Users role: Set up user accounts and home directories.
-# * Base-setup role: Set up hostname, mail forwarding, apt repositories, sshd,
-#   and sudo. Disable `cloud-init` if present to prevent it from overwriting
+# * Net: configure network interfaces.
+# * Resolv: configure the DNS resolver.
+# * Users: set up user accounts and home directories.
+# * Base-setup: set up hostname, mail forwarding, apt repositories, sshd, and
+#   sudo. Disable `cloud-init` if present to prevent it from overwriting.
 #   configurations.
-# * Net role: Configure network interfaces.
 ---
-- name: Run bootstrapping roles
-  hosts: '{{ target_hosts | d("all") }}'
-  become: true
+- name: Configure network interfaces
+  import_playbook: ./net.yaml
+
+- name: Configure the DNS resolver
+  import_playbook: ./resolv.yaml
   vars:
-    play_net_enable: '{{ net__enable | d(false) }}'
-    play_resolv_enable: '{{ resolv__enable | d(false) }}'
```

Comment:
I think `| d(false)` should be added in playbooks/resolv.conf, if it's going to be called from bootstrap? (as already done in playbooks/net.yaml)

Reply:
The thing is: I am not sure why those were there in the first place, as there are defaults for all those variables in `playbooks/group_vars/all/00-defaults.yaml`

Reply:
Ah. Perhaps [because they were created in ansible-roles repo first](https://github.com/Tina-pm/ansible-roles/pull/21/files), then moved here?

Reply:
ah, yes, that would explain it.

Reply:
I have removed the remaining `| d(false)` since it is not needed any more